tests/unit/modules/dcnm/fixtures/dcnm_policy_payloads.json

```diff
@@ -1537,5 +1537,30 @@
       "METHOD": "DELETE",
       "REQUEST_PATH": "https://10.64.78.151:443/rest/control/policies/POLICY-105105",
       "RETURN_CODE": 200
+    },
+
+    "config_preview": {
+      "RETURN_CODE": 200,
+      "METHOD": "GET",
+      "REQUEST_PATH": "https://10.195.225.193:443/appcenter/cisco/ndfc/api/v1/lan-fabric/rest/control/fabrics/mmudigon/config-preview?forceShowRun=false&showBrief=true",
```

Comment:
Better to use a private IP for the unit test.

Reply:
Done